assets/.eslintrc

```diff
@@ -0,0 +1,7 @@
+extends:
+  - '../.eslintrc'
+  - 'homeoffice/config/default'
+
+env:
+  es6: true
+  browser: true
```

Comment:
Is this what you meant about fixing the eslint config @lennym ? I moved it here because I think this config will only affect everything in the `assets/`directory

Reply:
YEAH!

Reply:
👍